packages/amplify-graphql-transformer-core/src/transformation/transform.ts

```diff
@@ -303,6 +308,18 @@ export class GraphQLTransform {
       }
       reverseThroughTransformers -= 1;
     }
+    // is this the right place?
+    for (const transformer of this.transformers) {
+      if (isFunction(transformer.getLogs)) {
+        const logs = transformer.getLogs();
+        if (logs.length) {
```

Comment:
Is this check required? I'd assume getLogs returns Log[], in which case you can just push the potentially empty array